src/test/scala/nl.knaw.dans.easy.deposit/StateManagerSpec.scala

```diff
@@ -104,11 +109,87 @@ class StateManagerSpec extends TestSupportFixture {
     }
   }
 
+  it should "mail the draft uuid" in {
+    draftPropsFile.writeText(
+      s"""state.label = SUBMITTED
+         |state.description = The deposit is ready for processing
+      """.stripMargin)
+    submittedPropsFile.writeText(
+      s"""state.label = REJECTED
+         |curation.performed = no
+         |state.description = rabarbera
+         |identifier.fedora = easy-dataset:1239
+      """.stripMargin)
+    StateManager(draftDeposit, submitBase, easyHome).getStateInfo should matchPattern {
+      case Success(StateInfo(State.submitted, """Something went wrong while processing this deposit. Please <a href="mailto:info@dans.knaw.nl?subject=%20%28reference%20nr:%20DRAFT/foo/7fa835ce-0987-4064-90ca-a7b75ce78a16%29&body=Hello%0A%0ACould%20you%20please%20figure%20out%20what%20went%20wrong%20with%20my%20deposit?%0A%0AIt%20has%20reference:%0A%20%20%20DRAFT/foo/7fa835ce-0987-4064-90ca-a7b75ce78a16%0Aand%20title:%0A%20%20%20%0A">contact DANS</a>""")) =>
+    }
+  }
+
+  it should "mail the submitted uuid" in {
+    draftPropsFile.writeText(
+      s"""state.label = SUBMITTED
+         |state.description = The deposit is ready for processing
+         |bag-store.bag-id = $submittedUuid
+      """.stripMargin)
+    submittedPropsFile.writeText(
+      s"""state.label = REJECTED
+         |curation.performed = no
+      """.stripMargin)
+    ((draftDeposit.bagDir / "metadata").createDirectories() / "dataset.json")
+      .write(
+        """{"titles":["
+          | A complete ground hog <a href='mailto:info@dans.knaw.nl'>weather</a> report
```

Comment:
The `easy-deposit-ui` happily writes HTML-tags in the `dataset.json`
It doesn't hurt plain text messages but I don't feel at ease.
Also: what happens when transformed to DDM and further down the line?

Reply:
Good point! Please make an issue for that or ask @rkruithof if he has already tested that...

Reply:
An altered test shows that DDM encodes the `<` as `&lt;`. So that part is safe.

Reply:
Now further trimming a title as of the first `<` within the 42 characters.